projects/smoke_pca9539r/src/main.c

```diff
@@ -0,0 +1,123 @@
+/* Smoke test for pca9539r gpio expander. i2c and other settings based off of
+   test_pca9539r_gpio_expander.c Program sets all pins as inputs, and low state, then outputs with
+   high state, and checks configuration registers against expected results Then toggles gpio state
+   on all pins every second */
```

Comment:
Generally we prefer a series of single-line `//` comments rather than block comments. Also, could you separate your sentences with periods?

Reply:
re: 'generally we prefer', it's actually in our (not often looked at) coding standards: https://uwmidsun.atlassian.net/wiki/spaces/ELEC/pages/9306218/Coding+Standards

Reply:
Interesting, searching `projects/*` for `//` gives 756 results, whereas searching for `*/` gives 15.

Reply:
Updated instructions, made some nice clean one line comments